spec/controllers/api/current/appointments_controller_spec.rb

```diff
@@ -90,5 +120,25 @@ def create_record_list(n, options = {})
         expect(response_facilities).not_to include(facility_in_another_group.id)
       end
     end
+
+    context 'handles appointment_type correctly' do
+      before :each do
+        set_authentication_headers
+        FactoryBot.create_list(:appointment, 10, facility: request_facility)
```

Comment:
Maybe just keep this list to `5`. Lot of our `create_list` tests build a lot of factories that slow the tests down.